src/api/users.js

```diff
@@ -17,4 +18,10 @@ export const getUsers = (page, params) => new Promise((resolve) => {
   }
 });
 
+export const getUserDetail = username => makeMockRequest({
+  uri: `/users/detail/${username}`,
+  method: METHODS.GET,
+  mockResponse: USER_PROFILE_MOCK[username],
```

Comment:
Please add an `errors` function checking if the username is in the USER_PROFILE_MOCK map